src/util.c

```diff
@@ -71,7 +71,7 @@ int hyper_list_dir(char *path)
 int hyper_copy_dir(char *src, char *dest)
 {
 	char cmd[512];
-	snprintf(cmd, sizeof(cmd), "/tar cf - -C %s . | /tar fx - -C %s", src, dest);
+	snprintf(cmd, sizeof(cmd), TAR_PATH" cf - -C %s . | " TAR_PATH" fx - -C %s", src, dest);
```

Comment:
Ouch, that's a problem. but as I said, I need defined XXXX_PATH, I don't want host path. I don't want the change of configure.ac.